File: exercises/beecrowd/1113/1113.go
// https://judge.beecrowd.com/pt/problems/view/1113
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)

	for scanner.Scan() {
		line := scanner.Text()
		parts := strings.Fields(line)
		if len(parts) != 2 {
			continue
		}

		x, _ := strconv.Atoi(parts[0])
		y, _ := strconv.Atoi(parts[1])

		if x == y {
			break
		}

		if x < y {
			fmt.Println("Crescente")
		} else {
			fmt.Println("Decrescente")
		}
	}
}

File: exercises/beecrowd/1113/1113.java
// https://judge.beecrowd.com/pt/problems/view/1113
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while (true) {
            int x = sc.nextInt();
            int y = sc.nextInt();

            if (x == y) {
                break;
            }

            if (x < y) {
                System.out.println("Crescente");
            } else {
                System.out.println("Decrescente");
            }
        }

        sc.close();
    }
}
